docs/configuration.asciidoc

```diff
@@ -666,6 +666,33 @@ while any positive integer value will be used as the maximum number of frames to
 To disable the limit and always capture all frames, set the value to `-1`.
 
 
+[float]
+[[config-span-stack-trace-min-duration]]
+==== `span_stack_trace_min_duration`
+
+<<dynamic-configuration, image:./images/dynamic-config.svg[] >>
+
+[options="header"]
+|============
+| Environment                                 | Django/Flask                    | Default
+| `ELASTIC_APM_SPAN_STACK_TRACE_MIN_DURATION` | `SPAN_STACK_TRACE_MIN_DURATION` | `"5ms"`
+|============
+
+By default, the APM agent will collect a stack trace with every recorded span
+which has a duration equal to or longer than this configured threshold.  While
```

Comment:
This sentence should use "that" instead of "which". The phrase, "_has a duration equal to or longer than this configured threshold_" is an essential clause, i.e. one that must be present for the sentence to be accurate. "which" should only be used to include information that is non-essential to the meaning of a sentence. 

```suggestion
By default, the APM agent collects a stack trace with every recorded span
that has a duration equal to or longer than this configured threshold.  While
```